onsite/lucxor/parallel.py

```
@@ -2,16 +2,538 @@
+        # Find spectrum in worker's local cache
+        spectrum = _mp_find_spectrum(
+            task.get('scan_num'),
+            task.get('rt', 0.0),
+            rt_tolerance
+        )
+
+        if spectrum is None:
+            return {'status': 'error', 'reason': 'spectrum_not_found', 'idx': task['idx']}
+
+        # Create Spectrum wrapper
+        spec_wrapper = Spectrum(spectrum, config)
```

Ruff (0.14.10)

Local variable `spec_wrapper` is assigned to but never used

Remove assignment to unused variable `spec_wrapper`

(F841)